go.mod

```diff
@@ -69,19 +69,19 @@ require (
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	gotest.tools v2.2.0+incompatible
-	helm.sh/helm/v3 v3.6.1
+	helm.sh/helm/v3 v3.7.2
```

Comment:
Is it necessary to upgrade these packages?

Reply:
Yes, version 3.6.1  package doesn't have this function.

Reply:
the ParseIntoString function?  it exists in 3.6.1. `vela install` also depends on this function.

Reply:
find it thanks。